docs/static/releasenotes.asciidoc

```diff
@@ -60,6 +61,47 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-17-2]]
+=== Logstash 7.17.2 Release Notes
+
+[[notable-7-17-2]]
+==== Notable issues fixed
+
+* Fixed an issue where the Plugin Manager would upgrade all dependencies in Logstash during an install/update operation of a single plugin. This often caused deployments to break due to unvetted 3rd party gem updates. https://github.com/elastic/logstash/pull/13804[#13804]
+* Fixed an issue where the GeoIP database lookup wasn't respecting the http_proxy environment variable https://github.com/elastic/logstash/pull/13841[#13841]
+* A couple of fixes improved the consistency of pipeline statistics in the HTTP API during pipeline startup https://github.com/elastic/logstash/pull/13845[#13845] and  https://github.com/elastic/logstash/pull/13850[#13850]
+* Logstash didn't correctly start on Windows if the folder containted spaces https://github.com/elastic/logstash/pull/13893[#13893]
```

Comment:
```suggestion
* Logstash now starts correctly on Windows, even if the folder contained spaces https://github.com/elastic/logstash/pull/13893[#13893]

Updates all Windows batch scripts used as CLI tools to quotes the %JAVACMD% to avoid path problems when the path contains spaces.


```

Reply:
We might not be able to state something so absolute. Still, it would be good to reword the item to focus on the fix rather than the problem.